Add grpc docs page (#803)
RoadieHQ/marketing-site: content/docs/details/grpc-docs/index.md

---
title: Use the grpc-docs plugin with your API entities
publishedDate: '2022-07-28T13:00:00.0Z'
description: How to use type:grpc-docs on your api entities

humanName: grpc-docs
integrationType: OSS plugin
---

# Introduction

> If you or your orgnaization uses gRPC you might be familiar with the [protoc-gen-doc](https://github.com/pseudomuto/protoc-gen-doc) plugin.
> 
> Roadie can render your protoc-gen-doc generated json files to give a better viewing experience in your api definition page.

Reviewer: ```suggestion
If your organization uses gRPC, you might be familiar with the [protoc-gen-doc](https://github.com/pseudomuto/protoc-gen-doc) plugin. Roadie can render your `protoc-gen-doc` generated JSON files to give a better viewing experience for your API definition page.
```

## Steps

> Configure your API entity files with a `type: grpc-docs` and point your `spec.definition` to the generated protoc-gen-doc json file.
> 
> That's it, now they will be rendered in your api definition page.

Reviewer: ```suggestion
To enable gRPC API docs, configure your API entity files with a `type: grpc-docs` and point your `spec.definition` to the generated `protoc-gen-doc json` file. Roadie will automatically pick up and render your API definition page.
```

> Example entity configuration

Reviewer: ```suggestion
Below is an example entity configuration for gRPC API docs.
```
